src/scripts.js

```diff
@@ -1,107 +1,316 @@
-// const chai = require("chai");
-// const expect = chai.expect;
-
-// const sampleData = require('../data/test-data');
-const prototypeRecipes = sampleRecipes;
-// const prototypeIngredients = recipeCards.sampleIngredients;
-const prototypeUsers = sampleUsers;
-const prototypeIngredients = sampleIngredients;
-// const prototypeUser1 = sampleData.sampleUsers[0];
-// const prototypeUser2 = sampleData.sampleUsers[1];
-
-// const Basket = require('./Basket');
-// const Ingredient = require('./Ingredient');
-// const Recipe = require('./Recipe');
-// const User = require('./User');
-// const Pantry = require('./Pantry')
+// -------------TEST_DATA-----------------------
+// const prototypeRecipes = sampleRecipes;
+// const prototypeUsers = sampleUsers;
+// const prototypeIngredients = sampleIngredients;
+
+const recipesData = recipeData;
+const ingredientsInfo = ingredientsData;
+const usersInfo = usersData;
```

Comment:
We learned a lot about accessing files (especially commenting out module.exports in the class files! lol!). It is all so much simpler than we realized. Thankfully!